src/bp-groups/bp-groups-template.php

```diff
@@ -3510,12 +3510,13 @@ function bp_group_member_promote_admin_link( $args = '' ) {
 	 */
 function bp_get_group_member_promote_admin_link( $args = '' ) {
 	global $members_template, $groups_template;
+	// error_log( print_r( groups_get_current_group(), true ) );
 
 	$r = bp_parse_args(
 		$args,
 		array(
 			'user_id' => ! empty( $members_template->member->user_id ) ? $members_template->member->user_id : false,
-			'group'   => &$groups_template->group,
+			'group'   => ! empty( $groups_template->group ) ? $groups_template->group : groups_get_current_group(),
```

Comment:
please revert this and whenever it's required please use the args of the function.